docs/database-engine/configure-windows/diagnostic-connection-for-database-administrators.md

```diff
@@ -36,58 +36,53 @@ ms.author: mikeray
-
+ The DAC is available and supported through the **sqlcmd** command-prompt utility using a special administrator switch (**-A**). For more information about using **sqlcmd**, see [Use sqlcmd with Scripting Variables](../../relational-databases/scripting/sqlcmd-use-with-scripting-variables.md). You can also connect prefixing **admin:** to the instance name in the format **sqlcmd -S admin:<*instance_name*>**. You can also initiate a DAC from a [!INCLUDE[ssManStudioFull](../../includes/ssmanstudiofull-md.md)] Query Editor by connecting to **admin:\<*instance_name*>**.  
+  
 ## Restrictions  
  Because the DAC exists solely for diagnosing server problems in rare circumstances, there are some restrictions on the connection:  
   
-- To guarantee that there are resources available for the connection, only one DAC is allowed per instance of [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)]. If a DAC connection is already active, any new request to connect through the DAC is denied with error 17810.  
+-   To guarantee that there are resources available for the connection, only one DAC is allowed per instance of [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)]. If a DAC connection is already active, any new request to connect through the DAC is denied with error 17810.  
   
-- To conserve resources, [!INCLUDE[ssExpress](../../includes/ssexpress-md.md)] does not listen on the DAC port unless started with a trace flag 7806.  
+-   To conserve resources, [!INCLUDE[ssExpress](../../includes/ssexpress-md.md)] does not listen on the DAC port unless started with a trace flag 7806.  
   
-- The DAC initially attempts to connect to the default database associated with the login. After it is successfully connected, you can connect to the master database. If the default database is offline or otherwise not available, the connection will return error 4060. However, it will succeed if you override the default database to connect to the master database instead using the following command:  
-
-   ```powershell
-   sqlcmd -A -d master 
-   ```
-
-   We recommend that you connect to the master database with the DAC because master is guaranteed to be available if the instance of the [!INCLUDE[ssDE](../../includes/ssde-md.md)] is started.  
+-   The DAC initially attempts to connect to the default database associated with the login. After it is successfully connected, you can connect to the master database. If the default database is offline or otherwise not available, the connection will return error 4060. However, it will succeed if you override the default database to connect to the master database instead using the following command:  
   
-- [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] prohibits running parallel queries or commands with the DAC. For example, error 3637 is generated if you execute either of the following statements with the DAC:  
+     **sqlcmd -A -d master**  

```

Comment:
This is a code block. It renders correctly in the current article. Not sure why we'd change to bold. 

I recognize that we've tagged the code-block as PowerShell and that sqlcmd is not PowerShell, but it it valid to run in PowerShell or other console. 

We will evaluate and may change to different type of code identifier, but not sure why we'd render it as bold text.